sdk/spring-sdk/src/test/java/kalix/springsdk/testmodels/view/ViewTestModels.java

```diff
@@ -92,6 +93,31 @@ public TransformedUser getUser(@RequestBody ByEmail byEmail) {
     }
   }
 
+  @Table("users_view")
+  public static class TransformedUserViewWithJWT extends View<TransformedUser> {
+
+    // when methods are annotated, it's implicitly a transform = true
+    @Subscribe.ValueEntity(UserEntity.class)
+    @JWT(
+        validate = JWT.JwtMethodMode.BEARER_TOKEN,
+        sign = JWT.JwtMethodMode.MESSAGE,
+        bearerTokenIssuer = {"a", "b"})
```

Comment:
I don't think JWT makes much sense on view transform methods, the update method will not be a public endpoint but something you feed with eventing